plugin/trino-elasticsearch/pom.xml

```diff
@@ -16,7 +16,7 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <dep.log4j.version>2.13.3</dep.log4j.version>
-        <dep.elasticsearch.version>6.0.0</dep.elasticsearch.version>
+        <dep.elasticsearch.version>6.7.0</dep.elasticsearch.version>
```

Comment:
Is the version bump required for this change? If not let's extract it to a separate change so that it can be merged faster. (Also we should make sure to not use the version of client which prevents communication with Open Distro for ES).

Reply:
I am using 'org.elasticsearch.common.time.DateFormatter' in Elasticsearch 6.7.0. We can extract it independently.

Reply:
You might find this change relevant then - https://github.com/trinodb/trino/pull/11263